### PR TITLE
Add simplified Jenkinsfile

### PR DESCRIPTION
**Pull request by damacus**
no Dockerfile changes
no added tests (actually removed the failing RabbitMQ tests)
removed the Docker tag from the make file, and moved this to the Jenkinsfile as this is the responsibility of Jenkins and never needs to be done accidentally locally by running `make build`

### CODE REVIEW

**Comment by damacus** [comment]
no Dockerfile changes
no added tests (actually removed the failing RabbitMQ tests)
removed the Docker tag from the make file, and moved this to the Jenkinsfile as this is the responsibility of Jenkins and never needs to be done accidentally locally by running `make build`

File: functions.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    docker_push "${REGISTRY:?}/${IMAGE_NAME}" "latest"
+  fi
+}
+
+# Semvertag doesn't like a 0 in the prefix right now
+# https://github.com/ministryofjustice/semvertag/issues/4
+# So strip 1604 from the tag
+# @args
+# $1 string
+strip_1604() {
```
Comment (dragoonis) [inline functions.sh]: this function appears to be unused (not called anywhere)

File: functions.sh [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  local STR
+  STR=${1//-1604/}
+  echo "${STR:?}"
+}
+
+# Remove a string within a given string (sed over variables)
+# @args
+# $1 Original string
+# $2 String to remove
+to_docker_tag() {
```
Comment (dragoonis) [inline functions.sh]: this function appears to be unused (not called anywhere)

File: functions.sh [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+# $1 PROJECT
+# $2 TAG
+push_image() {
+  local REGISTRY
+  local IMAGE_NAME=$1
+  local TAG=$2
+  REGISTRY="registry.service.opg.digital"
+  echo "Pushing image PROJECT:${IMAGE_NAME} TAG:${TAG}"
+  # We Docker always builds with a tag of latest
+  docker tag "${REGISTRY:?}/${IMAGE_NAME}:latest" "${REGISTRY:?}/${IMAGE_NAME}:${TAG}"
```
Comment (dragoonis) [inline functions.sh]: remove :? on the variable as it exists here

**Pull request merged**